### PR TITLE
Reworked helm chart for ThreatMapper

### PR DESCRIPTION
**Pull request by runitmisra**
Fixes # .

Changes proposed in this pull request:
-
-
-

@deepfence/engineering

### CODE REVIEW

File: deployment-scripts/helm-charts/deepfence-console-new/values.yaml [hunk 1]
```diff
@@ -0,0 +1,595 @@
...
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+  service:
+    type: ClusterIP
+
+sscanner:
```
Comment (ramanan-ravi) [inline deployment-scripts/helm-charts/deepfence-console-new/values.yaml:413]: Can you use full name `secretScanner`?

File: deployment-scripts/helm-charts/deepfence-console-new/values.yaml [hunk 2]
```diff
@@ -0,0 +1,595 @@
...
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+  service:
+    type: ClusterIP
+
+introuter:
```
Comment (ramanan-ravi) [inline deployment-scripts/helm-charts/deepfence-console-new/values.yaml:479]: Can you use full name `internalRouter`?

File: deployment-scripts/helm-charts/deepfence-console-new/values.yaml [hunk 3]
```diff
@@ -0,0 +1,595 @@
...
+
+  nodeSelector: {}
+
+  tolerations: []
+
+  affinity: {}
+  service:
+    type: ClusterIP
+
+pscaner:
```
Comment (ramanan-ravi) [inline deployment-scripts/helm-charts/deepfence-console-new/values.yaml:513]: Can you use full name `packageScanner`?

**Comment by runitmisra** [comment]
@ramanan-ravi I have removed the file, It was not being used

**Comment by runitmisra** [comment]
@ramanan-ravi @shahpratikr Can you please review this PR?

**Review by ramanan-ravi — APPROVED** [review]

**Pull request merged**